Stripe/STPAPIClient.m

```diff
@@ -52,7 +52,7 @@
 #import "STPCategoryLoader.h"
 #endif
 
-static NSString * const APIVersion = @"2019-05-16";
+static NSString * const APIVersion = @"2019-11-05";
```

Comment:
will revert this. I _think_ it's okay but I haven't tested fully and it's not needed